articles/access_controls.md

```diff
@@ -1,58 +1,57 @@
-{% include important.html content="Controlled data may not be redistributed." %}
-Controlled data can only be used by authorized investigators and are not transferable (unless explicitly specified otherwise). In other words, you cannot share Controlled data with collaborators unless they have been approved to use the data. Each Synapse users wishing to access Controlled Use data must individually agree to the `Conditions for Use` for that data.
-{% include important.html content="Investigators are responsible for determining the appropriate Conditions for Use for any data they upload into Synapse." %}
-It is important to note that `Conditions for Use` cannot be set for Synapse wiki or discussion forum. These are not designed to house data and therefore do not have local `Conditions for Use` as a feature. Synapse wikis and discussion forum inherit the condition for Use of their parent project.
+Some data requires data governance to appropriately manage its use. In Synapse this data is called `Controlled Data` and it is protected by `Conditions for Use`. `Conditions for Use` are typically applied in order to comply with the terms under which the data were collected or with other human subjects regulations. For example, human 'omic' data may have `Conditions for Use` imposed by informed consent requirements, legal contracts, or other privacy requirements. It is also appropriate to add `Condition for Use` to data collected from "vulnerable" populations and to content that could potentially harm individuals or groups if misused. 
 
-### Data that does not require Conditions for Use
-
-Open Data in Synapse are data without `Conditions for Use.` They can be used by any registered Synapse users authorized to view the project/folder/file/table (based on sharing setting) for research purpose. Typically Open Data are:
+{% include important.html content="`Controlled Data` may not be redistributed." %}
+`Controlled Data` can only be downloaded and used by authorized Synapse users. `Controlled Data` is not transferable unless explicitly specified otherwise. In other words, you cannot share `Controlled Data` with collaborators. Each Synapse user wishing to access `Controlled Data` must individually agree to the `Conditions for Use` to access that data.
 
-* Data from model organism, species or strain
-* Non-biological data such as presentations, results or images
+{% include important.html content="Synapse users are responsible for determining the appropriate `Conditions for Use` for any data they upload into Synapse." %}
+It is important to note that `Conditions for Use` cannot be set for Synapse `Wikis` or `Discussion Forums`: they are not designed to house data and therefore do not have local `Conditions for Use` as a feature. Synapse `Wikis` or `Discussion Forums` inherit the `Conditions for Use` of their Synapse parent `Project`.
 
```

Comment:
Disucssions and wiki's don't have conditions of use so can't inherit from them.  This is the confusion that came up during the discussion yesterday.